input/blog/2020-12-10-cake-vscode-extension-release-1.0.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cake VSCode Extension Release 1.0.0
+category: News
+author: gep13
+---
+
+The Cake Team are very happy to announce the 1.0.0 release of the [Cake Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=cake-build.cake-vscode).  This release has been a long time in the making, but we are hoping that it will be worth the wait!
+
+With the upcoming release of Cake 1.0.0, and the transition in our documentation to recommend the usage of the .Net Tool Runner for Cake, the main focus of this version of the extension was to start using the .Net Tool Runner for all of the built in actions.  For example:
+
+* If you choose `Run Task` from the command palette, the chosen task will be executed using the .Net Tool Runner for Cake
+* If you use the code lens options to either `run task` or `debug task`, the .Net Tool Runner for Cake will be used
+
+:::{.alert .alert-info}
+**NOTE:**
+
+If the .Net Global Runner for Cake isn't installed, it will be installed for you.
+:::
+
+In all of these cases, you can override the default options to revert to using either the .Net Framework or the .Net Core Runner for Cake.
+
+### Release Notes
+
+Check out the full set of [release notes](https://github.com/cake-build/cake-vscode/releases/tag/1.0.0) for this version.
+
+### Release Video
+
+There is a [short video](https://youtu.be/7Ba-pAHzO9w) discussing the main aspects of this release.
```

Comment:
Would it make sense to embed the video here instead of a link?

Reply:
I thought about it, but I wasn't sure if I should or not.  If you want me to, I am certainly happy to add this.

Reply:
I think I would embed it since it saves one click and increases possibility that people are watching it

Reply:
Done :+1: